src/firstvoices/backend/views/character_views.py

```diff
@@ -0,0 +1,65 @@
+from itertools import groupby
+from operator import itemgetter
+
+from django.db.models.functions import Upper
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from rest_framework.response import Response
+from rest_framework.viewsets import ModelViewSet
+
+from firstvoices.backend.models.characters import Character
+from firstvoices.backend.models.sites import Site
+from firstvoices.backend.predicates import utils
+from firstvoices.backend.serializers.character_serializers import CharacterSerializer
+from firstvoices.backend.views.base_views import FVPermissionViewSetMixin
+
+
+@extend_schema_view(
+    list=extend_schema(
+        description="A list of all characters the user currently has access to, organized by site.",
```

Comment:
I agree with this, I think I misunderstood the use case a bit with the list. I will re-work the list to return the list of characters based on a single site, rather than the sites a user has access to